packages/react-charts/src/components/ChartLegend/examples/ChartLegend.md

```diff
@@ -480,6 +421,234 @@ class InteractiveLegendChart extends React.Component {
 }
 ```
 
+```js title=Legend-tooltips
+import React from 'react';
+import { ChartLabel, ChartLegend, ChartPie, ChartThemeColor } from '@patternfly/react-charts';
```

Comment:
Add imports here.

Reply:
Ok, I assumed lint:md was run locally, as part of the build target? 

Fixed

Reply:
FYI, I intentionally added an error, but neither the `build` and `lint` targets pick it up. I have to run `lint:md` separately.

Reply:
Yeah, `lint:md` isn't a part of `lint`. We could add it to be now that `lint` doesn't take hours :)

Reply:
I should have waited to approve until the tests ran through fully, that's my bad... :(